scripts/testnet/include/nodes.sh

```diff
@@ -55,15 +55,19 @@ iterateOverObservers() {
   let "max_shard_id=$SHARDCOUNT - 1"
   for SHARD in `seq 0 1 $max_shard_id`; do
     for OBSERVER_IN_SHARD in `seq $SHARD_OBSERVERCOUNT`; do
-      $callback $SHARD $OBSERVER_INDEX
+      if [ $OBSERVER_INDEX -ne $SKIP_OBSERVER_IDX ]; then
+        $callback $SHARD $OBSERVER_INDEX
+      fi
       let OBSERVER_INDEX++
```

Comment:
If you want to replace `let` with `(( ))`, it needs to be replaced here as well. Back-ticks with `$()` too.

Reply:
Done, please check.

Reply:
Changed only in a few occurrences. Indeed, we should decide how to refactor all scripts